docs/release-notes.rst

```diff
@@ -197,6 +197,10 @@ Known Limitations
 
    * - Version
      - Description
+   * - 24.10.0
+     - | - There is a limitation when using NVIDIA NICs for a primary networks. We can't guarantee inbox or host-installed DOCA driver could be unloaded if it's used at the moment.
+           In a case if current driver unloaded successfully DOCA driver will restore only VFs configuration. VLAN and bonding and OpenVSwitch configuration won't re restored.
```

Comment:
Suggestion:

There is a limitation when using NVIDIA NICs for primary networks. We cannot guarantee that the inbox or host-installed DOCA driver can be unloaded if it is currently in use.

If the current driver is successfully unloaded, the DOCA driver container will restore only the VFs configuration; VLAN, bonding, and OpenVSwitch configurations will not be restored.